src/avram/tasks/db/migrations_status.cr

```diff
@@ -0,0 +1,42 @@
+require "shell-table"
+
+class Db::Migrations::Status < LuckyCli::Task
+  summary "Print the current status of migrations"
+
+  def call
+    if migrations.none?
+      puts "There are no migrations.".colorize(:green)
+    else
+      ensure_migration_tracking_tables_exist
+      print_migration_statuses
+    end
+  end
+
+  private def migrations
+    Avram::Migrator::Runner.migrations
+  end
+
+  private def ensure_migration_tracking_tables_exist
+    Avram::Migrator::Runner.new.setup_migration_tracking_tables
+  end
+
+  private def print_migration_statuses
+    puts ShellTable.new(
+      labels: ["Migration", "Status"],
+      label_color: :white,
+      rows: migration_statuses
+    )
+  end
+
+  private def migration_statuses
+    migrations.map do |migration|
+      status = if migration.new.migrated?
+                 "Migrated".colorize(:green)
```

Comment:
I think the yellow and green clash a bit. Could you leave this the default color?

Reply:
Fixed in https://github.com/luckyframework/avram/pull/135/commits/69e31114c0065ed2038b4f645e2f335da39a5823

Now looks like this:
<img width="479" alt="Screen Shot 2019-07-06 at 6 41 37 AM" src="https://user-images.githubusercontent.com/1279093/60756962-2466e500-9fb9-11e9-9424-300ad71f75d9.png">